client/collect.py

```diff
@@ -101,6 +101,7 @@ def run_file(run_path, run_name, results, parse_output=True,
 def run_dir(dir_path, parse_output=True, delete_after_success=False,
             submit_failures=False):
     results = {}
+    timeout = time.time() + 86400
```

Comment:
I like to make it easier on the next person by either:

1. Commenting when you throw a large int up there what it means (i.e., `# 86400 seconds = 24 hours`)
2. Using math to show your work (i.e., `time.time() + (24 * 60 * 60)`)

Regardless, see my comment below re: creating a relative time limit.